sql/postgresql/pggrammar/pggrammar.test/parsertest/aggregates.sql

```diff
@@ -0,0 +1,1218 @@
+--
```

Comment:
Move all example files from `sql/postresql/pggrammar/pggrammar.test/parsetest/*.sql` to `sql/postresql/examples`.